tests/integration/targets/docker_plugin/tasks/basic.yml

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
This file needs to be in the `tests/` subdirectory: tests/integration/targets/docker_plugin/tasks/tests/basic.yml